src/main/java/com/dre/brewery/configuration/configurer/Translation.java

```diff
@@ -12,8 +12,16 @@
 @Getter
 @AllArgsConstructor
 public enum Translation {
```

Comment:
Should we even use an enum here?
Maybe just do something like:
```java
private String translationsFile = "en.yml";

// ...

String getTranslationsFile() {
    return dir.resolve(Config.translationsFile);
}
```

Reply:
yeah, we could just swap it out. But honestly i don't mind the enum. Feels easier to work with than a String